include/fem-dem/gls_sharp_navier_stokes.h

```diff
@@ -524,13 +498,20 @@ Return a bool that describes  if a cell contains a specific point
 
   PVDHandler ib_particles_pvdhandler;
 
+
   const bool                   SUPG        = true;
   const bool                   PSPG        = true;
   const double                 GLS_u_scale = 1;
   std::vector<IBParticle<dim>> particles;
   double                       particle_residual;
 
   std::vector<TableHandler> table_p;
+
+  // Particles contact history
+
+  TableHandler table_residual;
```

Comment:
I'm not sure I understand. What is this table residual and what will it contain? Explain it a bit better please :).